.circleci/config.yml

```diff
@@ -32,11 +32,26 @@ jobs:
         chmod +x ./architect
         ./architect version
 
+    - run: |
+        date +"%Y" > /tmp/year
+        date +"%m" > /tmp/month
+        date +"%d" > /tmp/day
+    - restore_cache:
+        keys:
+        - go-cache-v1-{{ checksum "/tmp/year" }}-{{ checksum "/tmp/month" }}-{{ checksum "/tmp/day" }}
```

Comment:
Also why checksums? What is wrong with `go-cache-v1-2018-11-12`? What is v1?

Reply:
From https://circleci.com/docs/2.0/caching/#using-keys-and-templates, you can't use the value of an environment variable as part of a key, but you can use the checksum of the contents of a file, hence the checksum to get a date format for the key.

I've added `v1` in case we need to update the key format in the future (recommended in docs).